src/test/kotlin/study/Calculator.kt

```diff
@@ -0,0 +1,26 @@
+package study
+
+class Calculator {
+
+    fun calculate(expression: String): Int {
+        if (expression.isEmpty()) throw IllegalArgumentException("Input value is null or empty")
+        val tokenList = expression.split(" ").toList()
+        var result = tokenList[0].toInt()
+
+        for (i in 1 until tokenList.size - 1 step 2) {
+            if (tokenList[i].isEmpty() || tokenList[i + 1].isEmpty()) throw IllegalArgumentException("Input value is null or empty")
+            result = Calculator().executeOperation(
```

Comment:
```suggestion
            result = executeOperation(
```
Calculator는 없어도 되지 않을까요~?